Mathlib/Topology/Metrizable/Uniformity.lean

```diff
@@ -279,3 +279,16 @@ theorem UniformSpace.metrizableSpace [UniformSpace X] [IsCountablyGenerated (
   letI := UniformSpace.metricSpace X
   infer_instance
 #align uniform_space.metrizable_space UniformSpace.metrizableSpace
+
+lemma TotallyBounded.isSeparable {α : Type*} [UniformSpace α] [i : IsCountablyGenerated (𝓤 α)]
+    {s : Set α} (h : TotallyBounded s) : TopologicalSpace.IsSeparable s:= by
```

Comment:
Please reuse `{X : Type*}` from `variable`s above. Also, please mention `EMetric.subset_countable_closure_of_almost_dense_set` in the docstring. Otherwise LGTM. Thanks!
 
bors d+